packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -388,12 +392,25 @@ module.exports = {
                 docker: {
                   $ref: '#/definitions/docker',
                 },
+                buildFromSource: {
```

Comment:
Let's have this option inside docker section. There is a `definitions/dockerBuild`. Let's modify it. If path is set then we build from this path. What do you think?